### DISCUSSION TITLE
Compiling DB4S to use sqlite 3.46 when distro repos have sqlite 3.34 only?

### DISCUSSION THREAD

**Discussion by hchiper** [sqlitebrowser/sqlitebrowser#3685] [Q&A]
Hi all,

I have downloaded sqlitebrowser-3.13.x-rc1.tar.gz and I've been able to compile DB4S sucessfully on my system, after having installed the necessary dependencies.
DB4S runs well.
Unfortunately it is using sqlite3 3.34.1 (as mentioned in the "About" dialog). I was expecting sqlite3 3.46.0, as announced in the latest changelog.

I presume the reason is that I'm using a Linux distro (Q4OS 4) based on Debian Bullseye (oldstable) for which only version 3.34.1 of libsqlite3 and libsqlite3-dev are available in the repositories; so DB4S is compiled with these version.

On the other hand, I've been able to compile sqlite3 3.46, which runs fine.

Now I wonder if it would be possible to make a kind of merging, to have DB4S using sqlite 3.46.

Thanks in advance for any advice.

**Comment by lucydodo** [comment]
At this time, you'll need to build it yourself.
Please see the following documentation for how to build it: https://github.com/sqlitebrowser/sqlitebrowser/blob/master/BUILDING.md
As an aside, we will update the stable version to include the latest SQLite, thanks for letting us know. :)

**Reply by hchiper** [reply]
Thanks for your response.
I knew I needed to build it and that's what I did, with the result explained in my initial post.

**Reply by lucydodo** [reply]
Sorry, I was misunderstood.
You can change to SQLite prefix path with `cmake -DCMAKE_SYSTEM_PREFIX_PATH=<prefix> ..`

**Comment by hchiper** [comment]
Sorry for giving no feedback for one week.
I didn't realize I had to `sudo make install` after having compiled sqlite3 3.46 before compiling DB4S.
Now everything is OK. 
Thanks for your support.

**Reply by lucydodo** [reply]
You're welcome. :)
